web_tool_script_1.R

```diff
@@ -7,6 +7,7 @@ source("0_portfolio_input_check_functions.R")
 source("0_global_functions.R")
 source("0_web_functions.R")
 source("0_json_functions.R")
+source("0_portfolio_test.R")
```

Comment:
should this be `source("0_portfolio_input_check_functions.R")` since that's where you added the `pw_calculations()` function?

Reply:
no it also sources now a file from port_test ; portfolio_input_check_functions is already sourced!